modules/rosa-sdpolicy-am-cluster-self-service.adoc

```diff
@@ -16,4 +16,4 @@ Customers can self-service their clusters, including, but not limited to:
 * Add or remove machine pools and configure autoscaling
 * Define upgrade policies
 
-These tasks can be self-serviced using the `rosa` CLI utility.
\ No newline at end of file
+These tasks can be self-serviced using the {product-title} (ROSA) CLI (`rosa`) utility.
```

Comment:
"These tasks can be self-serviced" doesn't really make sense. "You can perform these self-service tasks" maybe?